### PR TITLE
add config files for micron+ (180mm) printer

### PR DESCRIPTION
**Pull request by Fragmon**

### CODE REVIEW

File: config/software/tilting/qgl_180mm.cfg [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+[quad_gantry_level]
+gantry_corners:
+   -60.2,-10.4
+   244.1,234.5
+points:
+   20,30
+   20,180
+   170,180
+   170,30
```
Comment (Frix-x) [inline config/software/tilting/qgl_180mm.cfg:23]: Same question about the points here: 180 seems high for a 180mm bed. Is this right?

**Review by Frix-x — CHANGES_REQUESTED** [review]
Hello,

Thank you for this PR. I'll just need some small changes to clean it a little bit and I'll merge it :)

**Comment by Fragmon** [comment]
I change the values and delete the #micron+

**Review by Frix-x — APPROVED** [review]
Super! Thanks for this, I'll merge it!

**Pull request merged**